### PR TITLE
fix(setup-wallet): dialogs and link

### PR DESCRIPTION
**Pull request by banklesss**
Related to:

- [YOMO-1304](https://emurgo.atlassian.net/browse/YOMO-1304)
- [YOMO-1306](https://emurgo.atlassian.net/browse/YOMO-1306)
- [YOMO-1307](https://emurgo.atlassian.net/browse/YOMO-1307)

[YOMO-1304]: https://emurgo.atlassian.net/browse/YOMO-1304?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[YOMO-1306]: https://emurgo.atlassian.net/browse/YOMO-1306?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[YOMO-1307]: https://emurgo.atlassian.net/browse/YOMO-1307?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by stackchain — APPROVED** [review]
move onto 4.27

File: apps/wallet-mobile/src/features/SetupWallet/useCases/CreateWallet/WalletDetailsScreen.tsx [hunk 1]
```diff
@@ -166,7 +165,26 @@ export const WalletDetailsScreen = () => {
...
+    strings.walletNameModalCardTitle,
+    strings.walletPasswordModalCardFirstItem,
+    strings.walletPasswordModalCardSecondItem,
+    strings.walletPasswordModalCardTitle,
+    styles.button,
+    styles.modal,
+  ])
+
+  React.useEffect(() => {
+    showModalTipsPassword()
```
Comment (michalszorad) [inline apps/wallet-mobile/src/features/SetupWallet/useCases/CreateWallet/WalletDetailsScreen.tsx]: Should we show this modal every time or only once?

File: apps/wallet-mobile/src/features/SetupWallet/useCases/CreateWallet/RecoveryPhraseScreen.tsx [hunk 2]
```diff
@@ -92,7 +92,27 @@ export const RecoveryPhraseScreen = () => {
...
+    strings.recoveryPhraseCardThirdItem,
+    strings.recoveryPhraseCardTitle,
+    strings.recoveryPhraseModalTitle,
+    styles.button,
+    styles.content,
+    styles.modal,
+  ])
+
+  React.useEffect(() => {
+    handleOnShowModal()
```
Comment (michalszorad) [inline apps/wallet-mobile/src/features/SetupWallet/useCases/CreateWallet/RecoveryPhraseScreen.tsx]: Should we show this modal every time the user visits this screen or only once?

File: apps/wallet-mobile/src/features/SetupWallet/useCases/CreateWallet/WalletDetailsScreen.tsx (hunk 1 above)
Comment (stackchain) [reply apps/wallet-mobile/src/features/SetupWallet/useCases/CreateWallet/WalletDetailsScreen.tsx]: by design it looks so, waiting for @banklesss confirmation.

**Comment by stackchain** [comment]
@banklesss conflicts, any feedback from Product?

**Comment by banklesss** [comment]
> any feedback from Product?

yes im applying a little change. Will be done in a minute

**Comment by banklesss** [comment]
done @stackchain

**Pull request merged**